src/pages/task/executionDropdown/ExecutionSelector.tsx

```diff
@@ -60,8 +61,9 @@ export const ExecutionSelect: React.VFC<ExecutionSelectProps> = ({
             <StyledTaskStatusIcon status={singleExecution.status} />
             <StyledP1>
               Execution {executionAsDisplay(singleExecution.execution)} -{" "}
-              {shortDate(
-                singleExecution.activatedTime ?? singleExecution.ingestTime
+              {getDateCopy(
+                singleExecution.activatedTime ?? singleExecution.ingestTime,
+                { tz }
```

Comment:
One of these values ends up undefined  when the page is loading and causes spruce to crash. https://dashboard.cypress.io/projects/yshv48/runs/3502/overview/6268fcc8-01c6-4c18-b402-8e1f7a3a03a7/video